src/trans/internal/tpm_fftw.F90

```diff
@@ -242,6 +242,7 @@ SUBROUTINE DESTROY_PLANS_FFTW
   IF( ASSOCIATED(TW%FFTW_PLANS) )  DEALLOCATE(TW%FFTW_PLANS)
   IF( ALLOCATED(TW%N_PLANS) )     DEALLOCATE(TW%N_PLANS)
 ENDIF
+TW%N_MAX=0
```

Comment:
That addition was definitely missing. Thanks for finding this @DJDavies2 !
I would just ask to insert the change instead within the preceding `IF( ASSOCIATED(TW )` statement.

Reply:
Thanks, I have done that.